tests/drizzle/test_kernel.py

```diff
@@ -0,0 +1,140 @@
+import pytest
+import numpy as np
+from astropy import wcs
+from drizzlepac import cdriz
+import cdriz_setup
+
+
+@pytest.fixture
+def kernel_pars():
+    _params = cdriz_setup.Get_Grid(inx=50, iny=60, outx=51, outy=66)
+    _params.zero_background()
+    return _params
+
+
+@pytest.mark.parametrize("kernel", ["square", "point", "turbo", "gaussian", "lanczos3"])
+def test_point_kernel(kernel, kernel_pars, new_truth=False, return_png=False):
+    """Function tests different c code point kernels (inputs already created on instantiation).
+
+    Parameters
+    ----------
+    kernel : str
+        String associated with one of the c code point kernel options.
+    kernel_pars : Class
+        The Class inintialized in Get_Grid which includes all of the inputs need to run cdriz.tdriz.
+    new_truth : bool, optional
+        Flag, whether to save a new csv truth file from the output data, by default False
+    return_png: bool, optional
+        Flag, whether to create an output image map (png).
+    """
+    # truth filename
+    truth_filename = f"./tests/drizzle/truth_files/{kernel}_truth"
```

Comment:
We probably should not hard-code specific filename paths.  Instead, construct the truth_filename by determining the installation directory (os.path.dirname(__file__) and then joining the installation directory with the specific kernel (whatever the current parameterized value is) and the literal text of the truth file ("_truth").  I think it will work out to something like I have noted here.

Reply:
good suggestions.

Reply:
I've added the local path to the save_array and save_fig functions.